phalcon/Storage/Adapter/Stream.zep

```diff
@@ -290,6 +295,17 @@ class Stream extends AbstractAdapter
         return this->getDir(key) . str_replace(this->prefix, "", key, 1);
     }
 
+    private function getIterator(string! dir) -> <Iterator>
```

Comment:
I am guessing this is working now with 0.12.0 Zephir. It was not working for the earlier version hence the `rglob` implementation.

Reply:
Offer to return `rglob`?

Reply:
No no, If this implementation works leave it. It is more modern and cleaner :)